src/Server.cc

```diff
@@ -160,6 +160,14 @@ Server::Server(const ServerConfig &_config)
     return;
   }
 
+  if (this->dataPtr->sdfRoot.WorldCount() == 0)
```

Comment:
Ah, this shouldn't have made it in gz-sim7 since it's now possible to load a model file.

Reply:
I will go ahead and add a PR to fix that so we don't lose it

Reply:
I cherry-picked it in here and squashed it down into the merge commit.